management/server/file_store.go

```diff
@@ -312,6 +344,42 @@ func (s *FileStore) GetAccountBySetupKey(setupKey string) (*Account, error) {
 	return account.Copy(), nil
 }
 
+// GetTokenIDByHashedToken returns the id of a personal access token by its hashed secret
+func (s *FileStore) GetTokenIDByHashedToken(token string) (string, error) {
+	s.mux.Lock()
+	defer s.mux.Unlock()
+
+	tokenID, tokenIDFound := s.HashedPAT2TokenID[token]
```

Comment:
It's not mandatory or even a recommendation, but usually, as check variable names mostly used `ok`.

Reply:
Thanks, will do that from now on.